sqlmesh/dbt/basemodel.py

```diff
@@ -283,7 +283,7 @@ def __init__(self, context: DbtContext, config: BMC):
         self._rendered_sql: t.Optional[str] = None
         self._enriched_config: BMC = config.copy()
 
-        self._jinja_globals = create_builtin_globals(
+        self._jinja_globals = sqlmesh.dbt.builtin.create_builtin_globals(
```

Comment:
why though?

Reply:
circular dependency

Reply:
I don't have strong opinions on the topic. Do you prefer the local import?

Reply:
yea just do a local import